### DISCUSSION TITLE
Congratulation!

### DISCUSSION THREAD

**Discussion by thirdygayares** [Jobboardplatform-svfc/JobBoardPlatform_Documentation#7] [Announcements]
Your group submitted in second place and some parts of your document are okay, but others need to be set aside for now You can choose one group activity from numbers 1 to 15, and I will add +20 points.

Also, expect that your system will collaborate with an Android app made by 3rd-year students. You will also work with backend developers and System administrators from BSIT 4B.

Reply to this discussion with the group activity you want to add.

Additional points promo expired at 6:00 am on October 1, 2024, if you did not reply

![image](https://github.com/user-attachments/assets/7ccb5ace-880c-4bd1-bee4-951e02d8b9b8)

**Comment by arjaycano** [comment]
Good am sir, my team chose group activity number 7 po.

**Reply by thirdygayares** [reply]
done
